Automattic/wp-calypso
# Offer Reset: include the PlansNavigation component in the Selector page

#### Changes proposed in this Pull Request

* Include the `PlansNavigation` bar at the top of the Selector page to display the current state of the cart and connect purchase/CTA buttons to the cart.

#### Testing instructions

* Run this PR.
* Make sure the Offer Reset AB experiment is on.
* Visit `http://calypso.localhost:3000/plans/:site`
* Verify that you see the `PlansNavigation` bar at the top of the page.
* Choose different plans/products and make sure you can purchase them.
* Verify that the bar reflects the real state of the cart.

Fixes 1169247016322522-as-1187323761633616

#### Demo
![image](https://user-images.githubusercontent.com/3418513/90438453-5dbdbf80-e0aa-11ea-9a55-a82e1c6bd35b.png)
